symbolic-unreal/Cargo.toml

```diff
@@ -27,7 +27,7 @@ serde = ["serde_", "chrono/serde"]
 
 [dependencies]
 anylog = "0.5.0"
-bytes = "0.4.12"
+bytes = "0.5.6"
```

Comment:
This will make it incompatible with Relay, can you check?

Reply:
yes, I think that needs updating too. Too bad I can’t use that git ref right now until we fix the breakpad submodule :-(

Reply:
relay already has both `0.4` and `0.5`, and the parameter of `symbolic-unreal` is not used anywhere.

Reply:
Thanks for verifying.